src/ecmult_impl.h

```diff
@@ -30,16 +30,24 @@
 #  endif
 #else
 /* optimal for 128-bit and 256-bit exponents. */
-#define WINDOW_A 5
-/** larger numbers may result in slightly better performance, at the cost of
-    exponentially larger precomputed tables. */
-#ifdef USE_ENDOMORPHISM
-/** Two tables for window size 15: 1.375 MiB. */
-#define WINDOW_G 15
-#else
-/** One table for window size 16: 1.375 MiB. */
-#define WINDOW_G 16
+#  define WINDOW_A 5
```

Comment:
Don't just document how the setting changes size in the configure script, document it in the code-- probably here-ish.  (Someone throwing out our build system will not find the docs inside configure. :) )